connectors/src/middleware/auth.ts

```diff
@@ -42,3 +56,37 @@ export const authMiddleware = (
   }
   next();
 };
+
+const _authMiddlewareWebhooks = (
+  req: Request,
+  res: Response<ConnectorsAPIErrorResponse>,
+  next: NextFunction
+) => {
+  if (req.path.startsWith("/webhooks")) {
+    const parts = req.path.split("/");
+    const indexOfWebhooks = parts.indexOf("webhooks");
```

Comment:
No way to use path templates here?

Reply:
Let’s just do “includes” no?

Reply:
Yes that's a lot simpler. I will do that.